### ISSUE TITLE
Which dataset is used for quantitative experiments？

### ISSUE THREAD

**Issue by HarshWinterBytes** [ShenhanQian/GaussianAvatars#74]
Thank you for such inspiring work!

I have some doubts about the quantitative experimental part of the paper, and I'd like to ask for your help in answering them:
1. The data in Table I are divided into two sections, one for Novel-View Synthesis and the other for Self-Reenactment. If these two parts correspond to the quantitative results on the validation and test dataset, respectively?
2. Is the fact that the experimental results of the Novel-View Synthesis are much better than those of the Self-Reenactment experiments due to the very severe overfitting of the 3DGS? Here are the results I reproduced on the validation and test dataset:
validation:
![image](https://github.com/user-attachments/assets/55a5a27a-d923-4b56-b9cf-fcaef6136241)
test:
![image](https://github.com/user-attachments/assets/02dd7e43-36ac-4d3b-978b-27d95621ba5f)

Looking forward to your early reply, best wishes!

**Comment by ShenhanQian** [comment]
Hi Chengyang,

1. Yes. You can find a direct reference of roles of these splits in this page: https://github.com/ShenhanQian/GaussianAvatars/blob/main/doc/offline_render.md
2. This large difference is mainly because we finetune FLAME parameters on the training set, which makes the performance on the validation set much better since they share the same set of FLAME parameters:  
    <img width="442" alt="image" src="https://github.com/user-attachments/assets/e872b786-e99b-4d65-98b3-a8ab1f56dae9">

**Comment by HarshWinterBytes** [comment]
Thank you very much for your early reply. I get it!!!
